src/puj/push/config.clj

```diff
@@ -0,0 +1,57 @@
+(ns puj.push.config
+  (:require [clojure.spec.alpha :as s]))
+
+
```

Comment:
These could use some comments. In particular, I'm not sure what ::growth is, and which one of these would be responsible for making sure a stack's contents (including exec) doesn't get too large during execution.

Reply:
Good catch. Added comments.